src/components/About.js

```diff
@@ -0,0 +1,17 @@
+import about_img_1 from "./assets/images/about_img_1.png";
+import about_img_2 from "./assets/images/about_img_2.png";
+import data from "./structure.json";
+
+function About() {
```

Comment:
More modern will be using the ES6 arrow function.
In the next [topic](https://www.robinwieruch.de/react-function-component/), you can receive some info about the arrow function in react.

Reply:
Done